Fabric.Authorization.AccessControl/src/index.html

```diff
@@ -9,6 +9,9 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <script>
     window.discoveryServiceRoot = "DISCOVERY_SERVICE_ROOT";
+
+    window.identityServiceRoot = "IDENTITY_SERVICE_ROOT";
```

Comment:
might have to change the authorization logic to insert empty strings and then look for empty strings instead